src/components/component.ts

```diff
@@ -234,6 +235,9 @@ export class Component {
    * Renders the Component without waiting for the next frame.
    */
   public renderImmediately() {
+    if (this._clipPathEnabled) {
+      this._updateClipPath();
+    }
```

Comment:
@jtlan this works by updating the clip path to match the current url every time it renders?

Why did clip path have to be dependent on the URL in the first place?